Mass: 36.1 # [kg]
Volume: 0.0386 # [m^3]
Ixx: 0.69 # [kg-m^2]
Iyy: 2.62 # [kg-m^2]
Izz: 2.51 # [kg-m^2]
Buoyancy_X_POS: 0.0205 # [m]
Buoyancy_Y_POS: 0.0 # [m]
Buoyancy_Z_POS: 0.04 # [m]
